addons/im_livechat/models/chatbot_script_step.py

```diff
@@ -31,6 +31,10 @@ class ChatbotScriptStep(models.Model):
         ('free_input_single', 'Free Input'),
         ('free_input_multi', 'Free Input (Multi-Line)'),
     ], default='text', required=True)
+    is_step_forward_operator = fields.Boolean(
+        string="Is Step Forward Operator",
+        compute="_compute_is_step_forward_operator"
+    )
```

Comment:
If we're getting short on time for the freeze (if it's not ready tomorrow basically), we might want to extract this change on a separate PR as this seems to be the only schema change and it can be done independently of the rest.

Reply:
Name suggestion: `is_forward_operator`
We are already on `step`, repeating it is not necessarily useful (for `step_type` it's useful because `type` alone is not a fun name to find in code)

Reply:
Prep PR for the new field: https://github.com/odoo/odoo/pull/190966